TWLight/applications/signals.py

```diff
@@ -248,11 +248,12 @@ def invalidate_bundle_partner_applications(sender, instance, **kwargs):
                 content_object=application,
                 site_id=site_id(),
                 user=twl_team,
+                # fmt: off
                 # Translators: This comment is added to open applications when a partner joins the Library Bundle, which does not require applications.
-                comment=_(
-                    "This partner joined the Library Bundle, which does not require applications."
+                comment=_("This partner joined the Library Bundle, which does not require applications."
                     "This application will be marked as invalid."
                 ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string